src/libstd/thread/mod.rs

```diff
@@ -713,22 +713,27 @@ struct Inner {
 
 #[derive(Clone)]
 #[stable(feature = "rust1", since = "1.0.0")]
-/// A handle to a thread.
+/// A handle to a thread, its just an abstract reference and as such
+/// it can be used to identify a thread (by name, for example). In most
+/// usage cases, this struct is not used directly.
```

Comment:
I am not totally sure what you mean by "used directly" here, could you maybe elaborate a bit on what you're thinking?